examples/twisted/websocket/echo_site_tls/server.py

```diff
@@ -47,10 +49,13 @@ def onMessage(self, payload, isBinary):
 
     log.startLogging(sys.stdout)
 
-    contextFactory = ssl.DefaultOpenSSLContextFactory('keys/server.key',
-                                                      'keys/server.crt')
+    cert = ssl.Certificate.loadPEM(open('keys/server.crt', 'r').read())
+    key = ssl.KeyPair.load(open('keys/server.key', 'r').read(), format=crypto.FILETYPE_PEM)
+    private_cert = ssl.PrivateCertificate.fromCertificateAndKeyPair(cert, key)
+    contextFactory = private_cert.options()
```

Comment:
FWIW, `certificateOptions` is the preferred naming for this, `contextFactory` is one of the things I search for when I audit a Twisted codebase :)